examples/slurm-mpi-hello/bootstrap-mpi.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Install OpenMPI packages from Ubuntu repos
+- hosts: all
+  become: true
+  tasks:
+  - name: install openmpi packages
+    apt:
```

Comment:
Would be good to use `package` if the package names are the same or have a RHEL section, but ok for now

Reply:
Annoyingly, the package names are different.

Also, the packages are built with slightly different options, and it's not clear if the RHEL packages include PMI support. This is relevant to the example because without PMI we need to manually supply a hosts file to `mpirun`.

I'm going to merge this as is for now, but note a follow-up to generalize this to RHEL.